### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by vchen30**
Thought these word choices will make your description sound better

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,13 +15,13 @@ Airtest
 Getting Started
 ---------------
 
-Airtest is a cross-platform automated testing framework focused mainly on games, but can also be used for native apps. Windows and Android are currently supported. iOS support will be coming in the near future.
+Airtest is a cross-platform automated testing framework focussing mainly on games, but can also be used for native apps. Windows and Android are currently supported; iOS support will be available in the near future.
```
Comment (Meteorix) [inline README.rst]: focusing?

File: README.rst [hunk 2]
```diff
@@ -43,13 +43,13 @@ Download AirtestIDE from our `homepage`_ if you need to use the GUI tool.
...
 
-You can find the complete airtest documentation on `readthedocs`_.
+You can find the complete Airtest documentation on `readthedocs`_.
 
 
 Example
 -------
 
-Airtest aims at providing platform independent API, so that you can write test once and run test on different devices.
+Airtest aims at providing platform independent API, so that you can write and run test on different devices.
```
Comment (Meteorix) [inline README.rst]: Here I mean `write once and run anywhere`.

**Comment by Meteorix** [comment]
@vchen30 Thanks for you PR! 
I left you some comments, please take a look.

File: README.rst (hunk 1 above)
Comment (adolli) [reply README.rst]: i think focusing is grammatically correct

**Pull request merged**